### ISSUE TITLE
Does not trigger re-render on props change

### ISSUE THREAD

**Issue by yuichkun** [phanan/vue-linkify#12]
## TL;DR

Hi, thank you very much for this awesome library!

Although this works with many cases, I found a situation where `v-linkified` component does not re-render after changes via props, data, computed, etc.

## Reproduction

Demo:
https://vue-linkify-not-working-demo.vercel.app/

Source Code:
https://github.com/yuichkun/repro-vue-linkify-bug

## Possible Fix?

I guess this bug derives from the fact that this library gets text values from `el.innerHTML`, so if there's a way to get text from `vNode`, it could avoid referencing the old value?? 

I'm not too familiar with Vue.js but if @phanan is busy, I'm up for trying making up a fix PR myself :)

**Comment by phanan** [comment]
You're always welcome to create a PR :)

Am Di., 23. März 2021 um 09:42 Uhr schrieb Yuichi Yogo <
***@***.***>:

**Comment by yuichkun** [comment]
@phanan 
Thank you!
Although I noticed that I have a slight misunderstanding in the first place, I did create a [PR](https://github.com/phanan/vue-linkify/pull/13), would you mind taking a look when you are free?
